k9mail/src/main/res/layout/message_view_attachment.xml

```diff
@@ -37,7 +37,7 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:ellipsize="middle"
-        android:maxLines="2"
+        android:singleLine="true"
```

Comment:
isn't this attribute deprecated?

Reply:
Yes it is deprecated, I just updated it. Thanks for spotting that, I was just assuming that the ellipsis would not work properly with maxLines attribute but it seems to work alright when it is a single line.

Reply:
Lots of issues with these attributes 🙃

Reply:
Google keeps changing their mind on APIs 🗡️